Dockerfile

```diff
@@ -56,13 +56,14 @@ RUN echo "@testing https://dl-cdn.alpinelinux.org/alpine/edge/main" | tee -a /et
         openssl-dev \
         openjdk21-jre \
 # Doc conversion
+		gcompat \
+    	libc6-compat \
         libreoffice \
 # pdftohtml
         poppler-utils \
 # OCR MY PDF (unpaper for descew and other advanced features)
         tesseract-ocr-data-eng \
 # CV
-        py3-opencv \
```

Comment:
`OpenCV not available in Python - Disabling OpenCV features: Extract Image Scans`

I think `extract-image-scans` is dependent

Reply:
Sorry yes need to readd
I can't build locally with CV there
Some weird issues

Reply:
I work with devcontainer or dockerfile.dev